policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/cost_of_living/base.yaml

```
@@ -0,0 +1,9 @@
+description: Quebec cost of living tax credit base amount.
```

Phrase others similarly
```suggestion
description: Quebec provides up to this base amount in its cost of living tax credit.
```